docs/docusaurus.config.js

```diff
@@ -43,8 +43,7 @@ module.exports = {
           items: [
             {
               label: 'Backend Docs',
-              href:
-                'https://github.com/anitab-org/anitab-forms-backend/wiki',
+              href: 'https://github.com/anitab-org/anitab-forms-backend/wiki',
```

Comment:
@nellyk I didn't see changes related to the referenced issue, I think this PR needs an update.

Reply:
@codesankalp  thank you for reviewing this, i raised this to address the current issue since the builds are failing. I wanted to raise a separe PR to address the issue permanently for that issue

Reply:
I will suggest going ahead in the same PR. It will be better to fix the failing build and the referenced issue in the same PR.
Just add the commits, it will be rebased when the PR is approved.

Reply:
great! i'll go ahead an add the changes here

Reply:
@codesankalp  i've updated the PR with the changes to resolve #156